chat-memories/spring-ai-jdbc-chat-memory/src/main/resources/org/springframework/ai/chat/memory/jdbc/schema-mariadb.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS ai_chat_memory (
+    conversation_id VARCHAR(36) NOT NULL,
+    content TEXT NOT NULL,
+    type VARCHAR(10) NOT NULL,
+    `timestamp` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT type_check CHECK (type IN ('USER', 'ASSISTANT', 'SYSTEM', 'TOOL'))
+);
+
+CREATE INDEX IF NOT EXISTS ai_chat_memory_conversation_id_timestamp_idx
+ON ai_chat_memory(conversation_id, `timestamp` DESC);
```

Comment:
Same comment as above.

Reply:
I updated the docs and specified that the chat memory table will be automatically created using autoconfiguration and how to disable it. I also removed the sql files that are not used.